packages/babel-plugin/src/__tests__/index.test.tsx

```diff
@@ -121,4 +121,55 @@ describe('babel plugin', () => {
       };"
     `);
   });
+
+  it('should not remove manual runtime import if no transformation occurs', () => {
+    const output = transformSync(
+      `
+      import { CC } from '@compiled/react/runtime';
+
+      <CC>
+        <div />
+      </CC>
+    `,
+      babelOpts
+    );
+
+    expect(output?.code).toMatchInlineSnapshot(`
+      "import { CC } from '@compiled/react/runtime';
+      <CC>
+              <div />
+            </CC>;"
+    `);
+  });
+
+  it('should append to manual runtime import if already present and transformation occurs', () => {
+    const output = transformSync(
+      `
+      import { CC as CompiledRoot, ax } from '@compiled/react/runtime';
+      import '@compiled/react';
+
+      const classes = ax['1', '2'];
```

Comment:
nit: invalid code lol

Reply:
Just for `ax` import 😅 its a function hehe lol